tests/dummy/lib/ember-blog/addon/routes/post.js

```diff
@@ -13,6 +13,20 @@ export default Ember.Route.extend({
   actions: {
     goToChineseVersion() {
       this.transitionTo({ queryParams: { lang: 'Chinese' } });
+    },
+    transitionToHome() {
+      this.transitionToExternal('home')
+        .then(() => {
+          var postController = this.controllerFor(this.routeName);
+          postController.set('transitionedToExternal', true);
```

Comment:
Thanks for reviewing.  I don't believe we can get the controller for `home` in the engine since `home` lives in the parent?